bin/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -Eeuxo pipefail
```

Comment:
What does this do? Maybe a comment?

Reply:
Default behaviour in bash is that a line with a failure doesnt break the script, but it carries on. With this set of flags, if there is an error during the script it will break; this minimise unexpected behaviours in the script during a failure (like a fail in the condition check that carries on and runs the next line).
So is a set of flags to break in condition errors (normal error, undefined variable,  pipe command error...)

Reply:
I saw it in the entrypoint that mysql uses to start, and I thought it was a good idea as a good practice to add it for every bash script.